css/src/components/icon.scss

```diff
@@ -0,0 +1,34 @@
+$icon-dimensions: 1.5rem !default;
+$icon-dimensions-small: 1rem !default;
+$icon-dimensions-medium: 2rem !default;
+$icon-dimensions-large: 3rem !default;
+
+.icon {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: $icon-dimensions;
+	height: $icon-dimensions;
```

Comment:
Could we remove the size modifiers and include SVG compat with something like:
```
.icon {
  display: inline-block;
  vertical-align: text-bottom;
  fill: currentColor;
  width: 1em;
  height: 1em;
}
```

Reply:
this is very nice, done